Hindi/hindi.md

```diff
@@ -0,0 +1 @@
+?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
```

Comment:
I haven't used hindi fonts, so I don't know whether it's possible to get GitHub to display them correctly. Sorry I can't help with this problem.